bioconda_utils/lint_functions.py

```diff
@@ -0,0 +1,186 @@
+import os
+import glob
+
+
+def _subset_df(recipe, meta, df):
+    """
+    Helper function to get the subset of `df` for this recipe.
+    """
+    name = meta['package']['name']
+    version = meta['package']['version']
+
+    # Some recipes have an empty build section, so it'll be None and we can't
+    # do a chained get.
+    build_number = 0
+    build_section = meta.get('build')
+    if build_section:
+        build_number = int(build_section.get('number', 0))
+    return df[
```

Comment:
Could the DF have a hierarchical index with name, version and build number, or is this operation to infrequent to justify this?